### PR TITLE
ipi-install-loki: install Loki via manifest sideloading

### PR DESCRIPTION
**Pull request by vrutkovs**
This would ensure that Loki is being installed early in the install process.

* Convert json manifests into YAML
* Copy Loki manifests to pre-install manifests dir, so that these would be applied on bootstrap instead of home-grown bash + oc post-install solution

This would allow us to collect logs from failed installs (assuming that Loki + promtail started and API server worked)

### CODE REVIEW

**Comment by vrutkovs** [comment]
Testplatfom issues were resolved

/retest

File: ci-operator/step-registry/ipi/install/loki/ipi-install-loki-commands.sh [hunk 1]
```diff
@@ -4,19 +4,15 @@ set -o nounset
...
-cat << EOF > /tmp/loki-manifests
----
+cat >> "${SHARED_DIR}/manifest_loki-ns.yml" << EOF
 apiVersion: v1
 kind: Namespace
 metadata:
   name: loki
----
+EOF
+cat >> "${SHARED_DIR}/manifest_loki-clusterrole.yml" << EOF
```
Comment (periklis) [inline ci-operator/step-registry/ipi/install/loki/ipi-install-loki-commands.sh:15]: I am missing the manifests moved into `${SHARED_DIR}` in this PR. Are they already there?

File: ci-operator/step-registry/ipi/install/loki/ipi-install-loki-commands.sh [hunk 2]
```diff
@@ -108,43 +113,43 @@ stringData:
...
-    max_look_back_period: 0
+      max_look_back_period: 0
     ingester:
-    chunk_block_size: 1572864
-    chunk_encoding: lz4
-    chunk_idle_period: 3m
-    chunk_retain_period: 1m
-    lifecycler:
-      ring:
+      chunk_block_size: 1572864
```
Comment (ingvagabund) [inline ci-operator/step-registry/ipi/install/loki/ipi-install-loki-commands.sh:120]: It'd rather have this in the previous commits where you convert JSON into YAML

File: ci-operator/step-registry/ipi/install/loki/ipi-install-loki-commands.sh (hunk 1 above)
Comment (vrutkovs) [reply ci-operator/step-registry/ipi/install/loki/ipi-install-loki-commands.sh:15]: Other steps may create additional manifests (i.e. [persistent metrics](https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/monitoringpvc/ipi-install-monitoringpvc-commands.sh))

File: ci-operator/step-registry/ipi/install/loki/ipi-install-loki-commands.sh (hunk 2 above)
Comment (vrutkovs) [reply ci-operator/step-registry/ipi/install/loki/ipi-install-loki-commands.sh:120]: I'll squash these two commits - neither can be applied by CVO separately anyway

**Comment by ingvagabund** [comment]
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/11731/rehearse-11731-pull-ci-openshift-cluster-authentication-operator-master-e2e-upgrade/1305802144858771456/artifacts/e2e-upgrade/gather-loki/ still produces loki tarball.

/lgtm
/hold

Keeping the hold in case other folks want to review as well. Feel free to cancel it.

**Comment by periklis** [comment]
@vrutkovs LGTM.

**Pull request merged**